scripts/xdebug.sh

```diff
@@ -17,6 +17,18 @@ if [ ! -f /etc/php.d/xdebug.ini ]
 then
     echo "[xdebug]
     zend_extension=\"/usr/lib64/php/modules/xdebug.so\"
+    xdebug.profiler_enable = 1
+    xdebug.profiler_enable_trigger = 1
+    xdebug.profiler_output_dir = \"/vagrant/profiling\"
     xdebug.remote_connect_back = 1
     xdebug.remote_enable = 1" > /etc/php.d/xdebug.ini
 fi
+
+if [ ! -d /vagrant/profiling ]; then
+	mkdir /vagrant/profiling
+fi
+
+#we need a bit longer to execute with profiling
+sed -i 's/max_execution_time.*/max_execution_time = 120/g' /etc/php.ini
```

Comment:
We should warn the user here that this parameter is being changed, as well as putting it in the docs.